thrift-definitions/exception/DbExceptions.thrift

```
@@ -0,0 +1,34 @@
+namespace java com.qreal.wmp.thrift.gen
+
+//Used in normal cases of excution (instead of returning null)
```

is this exception thrown always instead of returning null?

---

In search operations -- yes